### DISCUSSION TITLE
hping installation

### DISCUSSION THREAD

**Discussion by morevalar** [Homebrew/discussions#2183] [Everyday usage]
Hi! Is it possible to fix installation of hping on MAC OS X Mojave?

after installation there is two problems:
first is the symlink is unseen by the bash:

![2021-09-23_16-50-09](https://user-images.githubusercontent.com/19977456/134519350-e964db2f-85a7-46c0-92aa-2098d130c36d.png)

and the second is "Sorry, this hping binary was compiled without TCL scripting support" error message.
after you start it manually.

In the hping thread there is discussion and a possible solution. 
https://github.com/antirez/hping/issues/34

This bug exists  from 2018. Maybe it is time to fix it)))

**Comment by SMillerDev** [comment]
http://www.hping.org/download.php says that it's no longer being developed. But if you have fixes for it you're more than welcome to make a pull request.

**Comment by parsa222** [comment]
That error is because the homebrew-core formula built hping with --no-tcl
I made a tap that builds it with scripting on, feel free to use
```sh
brew install parsa222/tap/hping3
```
https://github.com/parsa222/homebrew-tap/blob/master/README.md
